purge.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python3
+""" purge.py
+    MediaWiki Action API Code Samples
+    Demo of `purge` module: Sending post request to purge cache of one page
+    MIT license
+"""
+import requests
```

Comment:
`Pylint` result looks good, true that! But, it might be nice to have blank lines in a few places like in [prefixsearch.py](https://github.com/srish/MediaWiki-Action-API-Code-Samples/blob/master/prefixsearch.py).

Reply:
Haha, Okay! I literally wanted the codes to be rated 10:00:00 on pylint, so I followed all the rules. 
Noted! I will add that soon

Reply:
@edyasikpo I see that you ended up adding only two blank spaces (on line 2 and 8). Do you see the blank lines in the sample code in [API:Documentation template](https://www.mediawiki.org/wiki/API:Documentation_template#Sample_code)? Ideally, we want to have blank spaces like that. The code is easier to read that way :-) Copying it below: 

```#!/usr/bin/python3

"""
    hello.py

    MediaWiki Action API Code Samples
    Demo of `...` module
    MIT license
"""

import requests

S = requests.Session()

URL = "https://en.wikipedia.org/w/api.php"

PARAMS = {
    'action':"query",
    'format':"json"
}

R = S.get(url=URL, params=PARAMS)
DATA = R.json()

print(DATA)
```
It looks like you've created a second pull request to add another file. Could you close that request and push the new changes here. Let me know if you've questions!